client/my-sites/site-settings/theme-setup/index.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import ActionPanel from 'my-sites/site-settings/action-panel';
+import ActionPanelTitle from 'my-sites/site-settings/action-panel/title';
+import ActionPanelBody from 'my-sites/site-settings/action-panel/body';
+import ActionPanelFooter from 'my-sites/site-settings/action-panel/footer';
+import ActionPanelFigure from 'my-sites/site-settings/action-panel/figure';
+import Notice from 'components/notice';
+import Button from 'components/button';
+import QueryActiveTheme from 'components/data/query-active-theme';
+import QueryTheme from 'components/data/query-theme';
+import { getSelectedSite } from 'state/ui/selectors';
+import { getActiveTheme, getTheme } from 'state/themes/selectors';
+import ActiveThemeScreenshot from './active-theme-screenshot';
+
+let ThemeSetup = ( { site, themeId, theme, translate, activeSiteDomain } ) => {
+	const onBack = () => {
+		page( '/settings/general/' + activeSiteDomain );
+	};
+
+	return (
+		<div className="main theme-setup" role="main">
+			{ site && <QueryActiveTheme siteId={ site.ID } /> }
+			{ themeId && <QueryTheme siteId={ 'wpcom' } themeId={ themeId } /> }
+			<HeaderCake onClick={ onBack }><h1>{ translate( 'Theme Setup' ) }</h1></HeaderCake>
+			<ActionPanel>
+				<ActionPanelBody>
+					<ActionPanelTitle>{ translate( 'Theme Setup' ) }</ActionPanelTitle>
+					<Notice status={ 'is-warning' } showDismiss={ false }>
+						{ translate( 'This action cannot be undone.' ) }
```

Comment:
Alternate string suggestion: 
* `translate( 'This cannot be undone' )`&mdash; translations: **3**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/settings-theme-setup)